{{ cookiecutter.package_name|slugify }}/docker-compose.yml

```diff
@@ -43,15 +39,15 @@ services:
     build:
       context: .
       target: app
-      {%- if cookiecutter.private_package_repository_name %}
-      secrets:
-        - poetry_auth
-      {%- endif %}
```

Comment:
Same comment: I think we need these build-time secrets here so that we can install private packages when building the app image.